requirements.txt

```diff
@@ -1,4 +1,3 @@
-Jinja2>=2.9.5
-MarkupSafe==0.23
+colorlog>=2.10.0
+jinja2>=3.0
 requests>=2.13.0
-colorlog==2.10.0
```

Comment:
MarkupSafe didn't seem to be used anymore, so I removed it (and loosened the pins on the other requirements)